src/views/CreateShareStructure.vue

```diff
@@ -1,18 +1,346 @@
 <template>
-   <section class="mt-10">
-      <header id="share-structure-header">
-        <h2>1. Create Your Authorized Share Structure</h2>
-      </header>
+  <section class="mt-10">
+    <header id="share-structure-header">
+      <h2>1. Create Your Authorized Share Structure</h2>
+    </header>
+    <p>
+      Add at least one class of shares. A share class consists of the name of the class, the maximum number of shares in
+      the class (including any associated Series), a par value for the class, and the currency the shares are valued in.
+    </p>
+
+    <!-- Help Section -->
+    <span class="help-btn" @click="helpToggle = !helpToggle">
+      <v-icon color="#1976d2" style="padding-right: 5px">mdi-help-circle-outline</v-icon>
+      <span v-if="!helpToggle">Help with Share Structure</span>
+      <span v-else>Hide Help</span>
+    </span>
+    <section v-show="helpToggle" class="share-structure-help">
+      <header id="share-structure-help-header"><h2>Share Structure Help</h2></header>
+      <p>
+        An incorporated business must issue shares. These shares represent ownership interest in the company and give
+        the shareholder a say in how the company is being run. For most small companies starting out, a simple share
+        structure with just one class of shares (and no series) is typical.
+      </p>
+      <p>
+        The sample below is for a typical share structure with just one class of shares. If there is more than one class
+        of shares, each class must be assigned an identifying name such as Class A, Class B, etc.
+      </p>
+      <div id="share-structure-sample">
+        <ListShareClass :shareClasses="sharesHelpSample"/>
+      </div>
+      <p><small>Sample Share Structure</small></p>
+      <h4>Important Information About the Company's Share Structure</h4>
+      <p>
+        The staff at the Corporate Registry cannot provide advice on how to set up your company's share structure. If
+        you do not understand what an authorized share structure is or what its purpose is or believe you need a more
+        complex share structure, you should seek professional advice or purchase an incorporation guide for detailed
+        information and infrastructure on establishing an authorized share structure.
+      </p>
+      <p>Refer to <a href="">helpLinkGoesHere</a> to obtain more information on incorporating a company</p>
+      <u class="help-btn" @click="helpToggle = !helpToggle"><small>Hide Help</small></u>
```

Comment:
ow are we tracking that this link needs to be updated? New ticket?

Reply:
BA team is yet to provide the link. But there is no ticket to track it

Reply:
Updated the link href with the value Sienna provided